### DISCUSSION TITLE
can't include zooz usb 500 stick and websocket 3000 not working

### DISCUSSION THREAD

**Discussion by drew571** [zwave-js/zwave-js-ui#2750] [Q&A]
I’m running Home Assistant on virtual box on an Intel NUC. I’ve been wanting to try out Z-Wave JS UI as opposed to Z-Wave JS for the added features and network maps…

I uninstalled the JS addon and installed the JS UI addon. I’m using a Zooz 500 series stick. I’m having two issues. This pops up in the Z-Wave JS UI control panel.

i've tried entering the usb stick name as the serial version and the version in the dropdown which is /dev/ttyACM0

neither one seems to work. 

I think this has to do with it not reading the usb stick?

The other issue I’m having is when I start the integration and point it to web socket and it fails to connect.

So far I’ve reset the stick. Restarted. Nothing seems to work. Any ideas would be much appreciated!

**Comment by drew571** [comment]
got it working.  multiple restarts.  just started working.

**Reply by robertsLando** [reply]
@drew571 How is the JS UI addon installed? If it's using docker it must be configured to use another port. A log would be useful here to understand what was going on

**Reply by drew571** [reply]
thanks for the response.  i did get it to see it and working.  i need to add devices to the network other than the usb controller but i think it's working.  it's not docker, it was installed via the addon store....

**Reply by robertsLando** [reply]
Ok cannot say what was the problem so, glad it works now :)
